app/views/files/files/_file.json.jbuilder

```diff
@@ -4,6 +4,7 @@ json.date file.created_at.strftime('%F %R')
 json.id file.id
 json.url file.link if file.link? && file.filetype.downcase.include?('google-apps')
 json.url file.foreign_ref if file.filetype == 'bruse/url'
+json.url file.foreign_ref if file.filetype == 'text/uri-list'
```

Comment:
Couldn't this be added as `if file.filetype == 'bruse/url || file.filetype == 'text/uri-list`?